src/main/java/cs2103/v15_1j/jimjim/ui/BottomPaneController.java

```diff
@@ -97,42 +105,32 @@ private void setUpCommandHistory(){
 	}
 
 	private void setUpHelpPopOver(){
-		this.helpPane = new GridPane();
-		helpPane.setHgap(10);
-		BorderPane.setMargin(helpPane, new Insets(SMALL_BORDER_WIDTH, SMALL_BORDER_WIDTH, SMALL_BORDER_WIDTH, 
-				SMALL_BORDER_WIDTH));
-
-		BorderPane helpPaneWrapper = new BorderPane();
-		helpPaneWrapper.setCenter(helpPane);
-
-		this.helpPopOver = new PopOver(helpPaneWrapper);
-		helpPopOver.setTitle("Help");
-		helpPopOver.setArrowLocation(PopOver.ArrowLocation.BOTTOM_RIGHT);
-		helpPopOver.getStyleClass().add("popover");
-
-		Label syntaxLabel = new Label("Syntax");
-		helpPane.add(syntaxLabel, 0, 0, 4, 1);
-
-		Label idLabel = new Label("{id} is listed next to the Task/Event in []");
-		helpPane.add(idLabel, 0, 1, 4, 1);
-
-		Label emptyLabel = new Label("");
-		helpPane.add(emptyLabel, 0, 2, 4, 1);
-
-		addHelpMessage("{name}", 0);
-		addHelpMessage("{name} by {date} {time}", 0);
-		addHelpMessage("{name} from {date} {time} to {date} {time}", 0);
-		addHelpMessage("delete {id}", 0);
-		addHelpMessage("mark {id} (as done)", 0);
-		addHelpMessage("unmark {id}", 0);
-		addHelpMessage("rename {id} to {name}", 0);
-		addHelpMessage("reschedule {id} to {date} {time}", 1);
-		addHelpMessage("extend {id} to {date}", 1);
-		addHelpMessage("search {filter}, {filter} ...", 1);
-		addHelpMessage("hide search", 1);
-		addHelpMessage("help", 1);
-		addHelpMessage("undo", 1);
-		addHelpMessage("redo", 1);
+		helpPopOverController = new PopOverController("Help", 4);
```

Comment:
Created a PopOverController to handle PopOvers instead of creating directly so that code can be reusable